src/components/Trade/hooks/useSwapper/utils.ts

```diff
@@ -166,6 +167,13 @@ export const getDefaultAssetIdPairByChainId = (
             buyAssetId: 'eip155:10/erc20:0x4200000000000000000000000000000000000042',
           }
         : ethFoxPair
+    case KnownChainIds.BnbSmartChainMainnet:
+      return featureFlags.BnbSmartChainZrx
```

Comment:
Missing flag in `FeatureFlags`

Reply:
https://github.com/shapeshift/web/pull/3947/commits/ce778b42ae0937dc6a41b9dccae7ef573cc47401